CHANGES.md

```diff
@@ -1,5 +1,88 @@
 # Changelog
 
+## 5.5 (2022-01-26)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/5.4...5.5)
+
+**FEATURE**
+
+ - **sct_deepseg:** Add models for MP2RAGE SC and MS lesion segmentation.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3637)
+ - **sct_testing:** Bring back `sct_testing` as a light wrapper for `pytest`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3605)
+ - **sct_deepseg:** Add spinal cord/gray matter multiclass segmentation model for 7T data.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3527)
+ - **sct_dmri_denoise_patch2self:** Add new Patch2Self CLI script for dMRI denoising.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3510)
+
+**CI**
+
+ - Switch to 'informational mode' to prevent Codecov failures for small changes.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3655)
+ - `tests.yml`: Update runners to no longer use Ubuntu 16.04.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3538)
+ - Add coverage via `codecov`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3463)
+
+**BUG**
+
+ - Add utility function to strip '.py' extensions.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3658)
+ - **sct_maths:** Appply -symmetrize on specified dimension in `sct_maths`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3657)
+ - **sct_compute_mtr:** Convert input file to float32 before compute MTR.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3638)
+ - **sct_get_centerline:** Make sure that the QC report for `sct_get_centerline` displays properly.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3626)
+ - Set `channel_axis=None` for `pyramid_expand` to fix bug in `skimage==0.19.0`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3616)
+ - **sct_propseg:** Fix argument parsing to ensure only the requested files are output.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3611)
+ - Restore the argparse linewrapping behavior we had prior to Python 3.7.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3604)
+ - **sct_download_data:** Update outdated mirror links.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3596)
+ - Upgrade to Qt5..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3588)
```

Comment:
Redundant period at the end, `Qt5..`, should be removed from the linked PR title.